frontend/src/app/home/home.page.html

```diff
@@ -20,7 +20,9 @@ <h3>We make the difference</h3>
     </div>
 
     <div class="button-container">
-      <button class="fancy-button">Apply now</button>
+      <button class="fancy-button" (click)="onStartApplication()">
```

Comment:
You might want to rename it to "startApplication" or "startJobApplication".

Reply:
This name still conveys the intended meaning. We have to discuss about naming conventions for properties, methods. We will first discuss and hear from every one. It is important to discuss ASAP before the code base becomes large. After discussion, If anythings needs to be changed I will change.

Reply:
"on" implies that something is already happening. But in your case, the application process has not started. It is the method that starts it.

This is even mentioned in the Angular Style Guide: https://angular.io/guide/styleguide#dont-prefix-output-properties

(I suggest reading the Style Guide in general. Has some valuable content.)

Reply:
So in most cases, I'd suggest sticking to the Style Guide or any other official documentation on the Angular website.